node/src/node.rs

```diff
@@ -66,14 +76,89 @@ where
             p2p: p2p.clone(),
         })?);
 
-        Ok(Node { p2p, syncer })
+        Ok(Node { p2p, store, syncer })
     }
 
+    /// Grab a reference to the [`P2p`] component of the node.
+    #[doc(hidden)]
```

Comment:
why hidden?

Reply:
do we want to expose them?

Reply:
In the Rust library I believe we should.

Reply:
exposed under `Node` with `request_` prefix as discussed offline